model_compression_toolkit/quantizers_infrastructure/keras/inferable_quantizers/base_uniform_inferable_quantizer.py

```diff
@@ -42,6 +41,9 @@ def __init__(self,
         """
         super(BaseUniformInferableQuantizer, self).__init__(quantization_target=quantization_target)
         self.num_bits = num_bits
+        assert np.all(max_range > min_range), f'Expected max_range to be bigger than min_range!'
+        min_range, max_range = adjust_range_to_include_zero(min_range, max_range, num_bits)
```

Comment:
Add warning if min_range or max_range changed because of this function.

Reply:
Ok but to avoid so many warning I set fix_range in QAT also before calling InferableQuantizer.